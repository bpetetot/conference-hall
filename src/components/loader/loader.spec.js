/* eslint-env jest */
import snap from 'tests/snapshot'
import { LoadingIndicator } from './index.js'

jest.mock('./loading', () => 'LoadingIndicator')
jest.mock('hoc-react-loader/build/core')

const snapshot = props => snap(LoadingIndicator)({ ...props })

describe('components/loader', () => {
  describe('LoadingIndicator', () => {
    it('should render', snapshot({ className: 'c' }))
  })
})
