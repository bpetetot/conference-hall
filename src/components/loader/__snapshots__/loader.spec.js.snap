// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/loader LoadingIndicator should render 1`] = `
<LoadingIndicator
  className="c"
/>
`;
